flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/core/testutils/CommonTestUtils.java

```diff
@@ -159,14 +159,14 @@ public static void setEnv(Map<String, String> newenv, boolean clearExisting) {
 
 	/**
 	 * A new object and the corresponding ClassLoader for that object, as returned by
-	 * {@link #createObjectFromNewClassLoader(ClassLoader)}.
+	 * {@link #createSerializableObjectFromNewClassLoader(ClassLoader)}.
```

Comment:
it _does_ create a serializable object, and I want to have a consistent naming pattern.

Reply:
IIRC it's also important for the using tests that this thing is actually serializable.

Reply:
I did not mean this particular method name `createSerializableObjectFromNewClassLoader`.
`ObjectAndClassLoader` is also returned by the exception version of `createObjectFromNewClassLoader`.
Ok, it is nit anyways: could add `as E.G. as returned by`
If `Serializab`ility is important, it could be also bounded type parameter: `T extends Serializable`

Reply:
I'm confused, and am not sure what exactly you want me to change. Do you want to change the signature of `createSerializableObjectFromNewClassLoader` or of `ObjectAndClassLoader`?

Reply:
note that I don't see the benefit in either of them. changing the signature of ´createSerializableObjectFromNewClassLoader` is basically a no-op, while changing `ObjectAndClassLoader` just makes it more restrictive without a benefit.

Reply:
It is only about jdoc for `ObjectAndClassLoader`.
`ObjectAndClassLoader` is returned from all methods: `createSerializableObjectFromNewClassLoader`, `createExceptionObjectFromNewClassLoader` and still `createObjectFromNewClassLoader` (now private).
`T extends Serializable` is just a thought, maybe not needed.